src/erlcloud_elb.erl

```diff
@@ -256,7 +439,7 @@ elb_simple_request(Config, Action, Params) ->
 
 elb_request(Config, Action, Params) ->
     QParams = [{"Action", Action}, {"Version", ?API_VERSION} | Params],
-    case erlcloud_aws:aws_request_xml4(get,
+    case erlcloud_aws:aws_request_xml4(post,
```

Comment:
This caused create_load_balancer() and creare_load_balancer_policy() to receive 400 Bad Request.